homeassistant/components/totalconnect/binary_sensor.py

```diff
@@ -129,7 +128,7 @@ async def async_setup_entry(
         for zone in location.zones.values():
             sensors.append(
                 TotalConnectZoneBinarySensor(
-                    coordinator, SECURITY_BINARY_SENSOR, location_id, zone
+                    coordinator, SECURITY_BINARY_SENSOR, zone, location_id
```

Comment:
Why swap the args?

Reply:
I'm guessing to align with `TotalConnectZoneEntity`

Reply:
I usually like to have the big objects first and then have the smaller properties like key or id at the end